src/gitHubIssueFetcher.ts

```diff
@@ -62,29 +75,61 @@ query findByLabel($queryString:String!, $pageSize:Int, $after:String) {
             endCursor
         }
     }
+    rateLimit {
+        limit
+        cost
+        remaining
+        resetAt
+    }
 }`;
 
 export class GitHubIssueFetcher {
-    private readonly logger: mariner.Logger;
+    private readonly logger: Logger;
 
-    public constructor(logger: mariner.Logger) {
+    public constructor(logger: Logger) {
         this.logger = logger;
     }
 
     public async fetchMatchingIssues(
         token: string,
         label: string,
         repositoryIdentifiers: string[]
-    ): Promise<IssueCountAndIssues> {
-        const listOfRepos = this.createListOfRepos(repositoryIdentifiers);
-        const variables: Variables = {
-            // NOTE: See https://docs.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests
-            queryString: `label:\"${label}\" state:open ${listOfRepos}`,
-            pageSize: 10,
-        };
-        const issueCountAndIssues = await this.fetchAllPages(token, query, variables);
+    ): Promise<GitHubIssue[]> {
+
+        const pageSize = 100;
+        const numberOfReposPerCall = 1000;
+        const chunks = this.splitArray(repositoryIdentifiers, numberOfReposPerCall);
```

Comment:
looking at this without context, I'm not sure what we're making a chunk of. Perhaps we should name chunks `chunksOf______` (fill in the blank with whatever it is). 

Or maybe it would be more clear if I understood what `repositoryIdentifiers` are? Is there a more clear name for that? I think changing this might be the most helpful way  to clarify things since it's used throughout this file.

Reply:
repositoryIdentifier is: owner/reponame. Which, for us, is a unique way to identify a specific repo. I'm open to other names, but nothing jumps to mind. 

As for the "chunksOf...", we could maybe say reposForEachCall. That makes it sound like it could be a map, but I suppose that's not terrible. It kind of is a map, but the keys are just the call numbers (0, 1, 2, ...). And the values are arrays of repo identifiers.

Reply:
Ok, if it's owner/reponame, I might call it repositoryFullName or repositoryNameForURL. If you don't like those, we can keep it for now and if I come up with something brilliant in the future we can change it then. 

I like reposForEachCall.